app/src/main/res/layout/fragment_autofill_onboarding.xml

```diff
@@ -0,0 +1,140 @@
+                android:fontFamily="sans-serif"
+                android:textStyle="normal"
+                android:textColor="@color/text_grey"
+                android:lineSpacingExtra="6sp"
+                android:gravity="center_horizontal"
+                android:text="@string/onboarding_autofill_title"
+                android:layout_marginTop="30dp"
+                android:layout_marginStart="43dp"
+                android:layout_marginEnd="43dp"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+        />
+
+        <TextView
+                android:id="@+id/autofillDescription"
+                android:layout_width="310dp"
+                android:layout_height="wrap_content"
+                android:textSize="15sp"
+                android:fontFamily="sans-serif"
+                android:textStyle="normal"
+                android:textColor="@color/text_grey"
+                android:lineSpacingExtra="5sp"
+                android:gravity="center_horizontal"
+                android:layout_marginTop="12dp"
+                android:layout_marginStart="25dp"
+                android:layout_marginEnd="25dp"
+
+                android:text="@string/onboarding_autofill_description"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/autofillTitle"
+        />
+
+        <Button
+                android:id="@+id/goToSettings"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:text="@string/go_to_settings"
+                android:textColor="@color/text_white"
+                android:backgroundTint="@color/color_primary"
+                android:layout_marginBottom="31dp"
+                android:textSize="14sp"
+                android:fontFamily="sans-serif-medium"
+                android:textStyle="normal"
+                android:letterSpacing="0.09"
+                android:lineSpacingExtra="2sp"
+                android:gravity="center_horizontal|center_vertical"
+                android:background="@drawable/rounded_corner_selector"
+                android:textAllCaps="true"
+                android:elevation="1dp"
+                android:layout_marginTop="70dp"
+                android:layout_marginStart="90dp"
+                android:layout_marginEnd="90dp"
+                android:paddingEnd="18dp"
+                android:paddingStart="18dp"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/autofillDescription"
+                app:layout_constraintBottom_toTopOf="@id/skipOnboarding"
+
+        />
+
+        <Button
+                android:id="@+id/skipOnboarding"
```

Comment:
naming nit: doesn't this just let you skip the _screen_, not skip onboarding all together? I'd probably just name this `skip` or `skipButton`